pantos/validatornode/celery.py

```diff
@@ -36,12 +37,15 @@ def is_main_module() -> bool:
     _logger.info('Initializing the Celery application...')
     initialize_application()  # pragma: no cover
 
+ca_certs = {} if config['celery']['broker'].startswith('amqp') else {
```

Comment:
Is this config variable mandatory? If not we will have to check if it exists I guess

Reply:
Yes, the config value celery.broker contains the URL of the message broker.